Build failure: gbenchmark

### Steps To Reproduce

Steps to reproduce the behavior:
1. build *X*

### Build log

`nix log /nix/store/z0wmd27qjgnjri5fm9h1jjy4lg37v518-gbenchmark-aarch64-unknown-linux-gnu-1.9.0.drv`

```
....

CMake Warning at cmake/CXXFeatureCheck.cmake:49 (message):
  If you see build failures due to cross compilation, try setting
  HAVE_PTHREAD_AFFINITY to 0
Call Stack (most recent call first):
  CMakeLists.txt:323 (cxx_feature_check)


-- Performing Test HAVE_PTHREAD_AFFINITY -- success
CMake Error at CMakeLists.txt:343 (find_package):
  Could not find a package configuration file provided by "GTest" with any of
  the following names:

    GTestConfig.cmake
    gtest-config.cmake

  Add the installation prefix of "GTest" to CMAKE_PREFIX_PATH or set
  "GTest_DIR" to a directory containing one of the above files.  If "GTest"
  provides a separate development package or SDK, be sure it has been
  installed.
```

### Additional context

cross compiling from x86_64-linux to aarch64

### Notify maintainers

@emilazy @alyssais 


### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
✦ ❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.1, NixOS, 24.11 (Vicuna), 24.11.20241009.5633bcf`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.9`
 - channels(root): `"nixos-22.05"`
 - nixpkgs: `/nix/store/sdzpqjwx7pdx6lsq6llyfqqf7hspp83c-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/libraries/gbenchmark/default.nix
+++ b/pkgs/development/libraries/gbenchmark/default.nix
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ninja ];
 
-  checkInputs = [ gtest ];
+  buildInputs = [ gtest ];
 
   cmakeFlags = [
     (lib.cmakeBool "BENCHMARK_USE_BUNDLED_GTEST" false)

